index.md

```diff
@@ -9,8 +9,20 @@ so this introduction to Python is built around a common scientific task:
 **data analysis**.
 
 ### Arthritis Inflammation
-We are studying **inflammation in patients** who have been given a new treatment for arthritis, and
-need to analyze the first dozen data sets of their daily inflammation. The data sets are stored in
+We are studying **inflammation in patients** who have been given a new treatment for arthritis. 
+
+There are 60 patients, who had their inflammation levels recorded for 40 days. We want to analyze these recordings to study the effect of the new arthritis treatment. 
```

Comment:
Thanks for shortening the lines mentioned. Could you shorten this line as well to keep it under 100 characters?